exampleCourse/questions/demo/surveyQuestion/question.html

```diff
@@ -0,0 +1,47 @@
+<pl-question-panel>
+
+    <p>Please answer the following demographic survey questions. </p>
+    <p>As this is a survey, there are no correct or incorrect responses.</p>
+
+    <hr>
+
+    <h6>Race/Ethnicity: Check all that apply</h6>
+
+    <p><input type="checkbox" name="race" value="white" id="1" /> <label for="1">White or European American</label></p>
```

Comment:
HTML ids cannot start with a digit. Note that you can use label around the input, in that case an id is not necessary:

```suggestion
    <p><label><input type="checkbox" name="race" value="white" /> White or European American</label></p>
```

Reply:
This totally slipped my mind, great catch! Updated here https://github.com/PrairieLearn/PrairieLearn/pull/1867/commits/9d370d64f0be83c6055411ef822b2db5a6903043